From 5a8289ba706c00845fc6700b614dc8b7d4a4a836 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 17 Oct 2023 13:57:51 -0700
Subject: [PATCH] Workaround bug with wheel creation

/usr/bin/salt is attempted to be installed more than once causing
build failures, temporary workaround while upstream fixes.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index c299254..06bc78d 100755
--- a/setup.py
+++ b/setup.py
@@ -1033,6 +1033,7 @@ class SaltDistribution(distutils.dist.Distribution):
     @property
     def _property_scripts(self):
         # Scripts common to all scenarios
+        return []
         scripts = ["scripts/salt-call"]
         if self.ssh_packaging or PACKAGED_FOR_SALT_SSH:
             scripts.append("scripts/salt-ssh")
-- 
2.42.0

